tests/pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+#base_url = https://netpunkt-mockup-develop.builds.dbc.dk
```

Comment:
Skulle vi ikke have bibdk her i stedet for netpunkt - selvom det er ukommenteret?